### PR TITLE
Fix warnings in measurement_container tests

### PR DESCRIPTION
**Pull request by leokoppel**
Minor changes. No behaviour change.

Part of #41

### CODE REVIEW

File: wave_containers/tests/containers/measurement_test.cpp [hunk 1]
```diff
@@ -133,14 +132,14 @@ TEST(Utils_measurement, iterators) {
     for (auto i = 0; i < 5; ++i) {
         auto now = std::chrono::steady_clock::now();
         auto value_in = i * 1.1;
-        m.emplace(now, SomeSensors::S1, i);
+        m.emplace(now, SomeSensors::S1, value_in);
```
Comment (navganti) [inline wave_containers/tests/containers/measurement_test.cpp]: Just curious, value_in != i. Why are you multiplying by 1.1?

Comment (leokoppel) [inline wave_containers/tests/containers/measurement_test.cpp]: You are right. The test had two slight bugs in it which cancelled out.

Comment (navganti) [inline wave_containers/tests/containers/measurement_test.cpp]: Ah okay. LGTM

Comment (navganti) [inline wave_containers/tests/containers/measurement_test.cpp]: Ah okay. LGTM.

Comment (chutsu) [inline wave_containers/tests/containers/measurement_test.cpp]: what? bugs in Leo's code? no way!

**Pull request merged**